BuildResidentialHPXML/measure.rb

```diff
@@ -1766,7 +1772,11 @@ def arguments(model)
     arg.setDisplayName('Water Heater: First Hour Rating')
     arg.setDescription("Rated gallons of hot water supplied in an hour. Required if Efficiency Type is UniformEnergyFactor and Type is not #{HPXML::WaterHeaterTypeTankless}. Does not apply to space-heating boilers.")
     arg.setUnits('gal/hr')
-    arg.setDefaultValue(56.0)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('water_heater_usage_bin', water_heater_usage_bin_choices, false)
+    arg.setDisplayName('Water Heater: Usage Bin')
+    arg.setDescription("The usage of the water heater. Required if Efficiency Type is UniformEnergyFactor and Type is not #{HPXML::WaterHeaterTypeTankless}. Does not apply to space-heating boilers.")
```

Comment:
If you’re adding this argument, then remove the FHR one above.

Reply:
Can we not have the flexibility of providing either option?

Reply:
They are both equally available inputs (on the EnergyGuide label), so there’s really no advantage to that.

Reply:
Which of the two do you think we should provide?

Reply:
I'd recommend providing just Usage Bin (which can be defaulted in OS-HPXML).